extensions/amp-auto-ads/0.1/placement.js

```diff
@@ -216,7 +216,7 @@ export class Placement {
           return this.state_;
         }
         const useResponsiveAdElement =
-          isResponsiveEnabled && this.responsiveSizingBranch_ == '368226531';
+          isResponsiveEnabled && this.responsiveSizingBranch_ != '368226533';
```

Comment:
can we import the value? easy to forgot updating this line later

Reply:
Not easily - importing it seemed to create a circular dependency.

Reply:
then the value declaration should be moved here?